uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -39,10 +38,18 @@
     padding-top: 15px;
 
     @media screen and (max-width: $screen-md-max) {
-      > div {
+      > div:not(:last-child) {
         padding-bottom: 20px;
       }
     }
+
+    @media screen and (min-width: $screen-md-min) and (max-width: $screen-md-max){
+      > div:nth-child(4),
+      > div:nth-child(5),
+      > div:nth-child(6) {
```

Comment:
This could be very fragile, what would happen if an extra div element is added? A class should be added for these divs.